easybuild/easyconfigs/q/Qwt/Qwt-6.1.5-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'Qwt'
+version = '6.1.5'
+
+homepage = 'http://qwt.sourceforge.net/'
```

Comment:
Can you use `https` instead of `http`?

Reply:
Resolved in https://github.com/easybuilders/easybuild-easyconfigs/pull/11064/commits/ae8e720b1f4a8af39096f0216294be2d416d7aef 

http://qwt.sourceforge.net/ ->  https://qwt.sourceforge.io/

Reply:
In fact, the homepage should be changed to `https://qwt.sourceforge.io/` (the `.net` one doesn't work with `https`, but it auto-redirects to `https://qwt.sourceforge.io/`